### ISSUE TITLE
fix noinspection unchecked for excluded CheckBox in GuiConfigController

### ISSUE THREAD

**Issue by tarheel** [BrightSpots/rcv#304]

**Comment by HEdingfield** [comment]
This applies to `GuiConfigController.initialize()`:
```
// noinspection unchecked
return new SimpleObjectProperty(checkBox);
```

**Comment by HEdingfield** [comment]
I believe this was also resolved with #462.

**Comment by HEdingfield** [comment]
The "fix" in #462 resolved the warning, but it also broke checkboxes in the Candidates table (now they only appear as "true" or "false" and it's uneditable. Need to roll this back.

**Comment by HEdingfield** [comment]
Duplicate of #280.
